addons/road-generator/road_point.gd

```diff
@@ -508,3 +530,72 @@ func _is_junction_valid(point: RoadPoint)->bool:
 		if next_point == self:
 			return true
 	return false
+
+## If one RoadPoint references the other, but not the other way around,
+## but itself has an empty slot in the right "orientation", then we assume
+## that the user is manually connecting these two points, and we should finish
+## the reference by making the reference bidirectional.
+##
+## TODO: Technically need to understand the case whereby someone is trying
+## to *clear* the reference to another roadpoint.
+##
+## Returns true if any updates made, false if nothing changed.
+func _autofix_noncyclic_references(
+		old_point_path: NodePath,
+		new_point_path: NodePath,
+		for_prior: bool) -> void:
```

Comment:
Any suggestions for a clearer variable name? I agree with you it's not that clear. Will add a description either way.

The meaning is in essence to say if we're doing the prior or next point's validation fix. So it's a flag which makes me think bool, rather than trying to artificially use one of our existing ENUMs which I think would just make it more confusing.